views/address.tmpl

```diff
@@ -173,11 +179,6 @@
                 {{end}}
 
                 {{if lt .NumFundingTxns .KnownFundingTxns}}
-                {{if not .Fullmode}}
-                <div>
-                    *1000 is the upper limit for total number of transactions shown
```

Comment:
This needs to stay.  The idea of the superscript asterisk is to indicate that there is a footnote to read.  After this, we're all set.  :smile: